lib/access_import/acc_utils.rb

```diff
@@ -0,0 +1,54 @@
+module AccUtils
+  def make_mappable(table_name, primary_key, sanitize = false)
+    @acdb[table_name].map do |r|
+      [
+        r[primary_key].to_i,
+        sanitize ? sanitize_record(r) : r
+      ]
+    end.to_h
+  end
+
+  def down_hkeys(row)
+    row.keys.map do |key|
+      [key.to_s.underscore.to_sym, row[key]]
+    end.to_h
+  end
+
+  def parse_int_fields(record, keys)
+    keys.each do |key|
+      record[key] = record[key].to_i if record[key]
+    end
+    record
+  end
+
+  def parse_datetime_fields(record, keys)
+    keys.each do |key|
+      record[key] = record[key].to_datetime if record[key]
+    end
+    record
+  end
+
+  def parse_date_fields(record, keys)
+    keys.each do |key|
+      record[key] = record[key].to_date if record[key]
+    end
+    record
+  end
+
+  def salutation(anrede)
+    return 'mrs' if anrede == 'Frau'
+    return 'mr' if anrede == 'Herr'
+    nil
+  end
+
+  def email(h_email)
+    return "unknown_email_#{Time.now.to_i}@example.com" if h_email.nil?
+    h_email.sub(/^\#mailto:/, '').sub(/\#$/, '')
+  end
+
+  def map_gender(value)
+    return nil unless value
+    return 'female' if value == 'W'
```

Comment:
yes I understand that when you implemented this, this was still included, but I don't agree with B. Implementing code for the sake of "maybe we could use it in the future" is in my opinion not optimal and this causes again more inconsistency. 
Removing, adding back, removing or refactoring it one more time is just a back and forth ;)
(And it was an acceptance criteria/requested from them to remove the gender)